cpp/src/kalman_filter/lkf.h

```diff
@@ -253,6 +253,7 @@ void find_kalman_gain(Variables<T> &var, cublasHandle_t handle,
   CUBLAS_CHECK(cublasgemm(handle, CUBLAS_OP_N, CUBLAS_OP_N, var.dim_z,
                           var.dim_z, var.dim_x, &alfa, var.H, var.dim_z, var.K,
                           var.dim_x, &beta, var.R_cpy, var.dim_z, stream));
+  CUDA_CHECK(cudaStreamSynchronize(stream));
```

Comment:
This is pretty expensive. Are these really necessary or we’re you using them just to debug?

Reply:
Forgot about the first commit that was about debugging indeed!